.github/workflows/release_linux_x86_64.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         python -m pip install -q -r requirements-release.txt
 
     - name: Build manylinux2014_x86_64
-      uses: docker://quay.io/pypa/manylinux2014_x86_64
+      uses: docker://quay.io/pypa/manylinux2014_x86_64:latest@sha256:085b7c2c1bcb45b936e5478fe24a5142b0519e242a0104142fd50d569d6cb922
```

Comment:
I am inclined to always use the latest docker image. Will [dependabot](https://github.com/apps/dependabot) also help us to bump the version if there is an update?

Reply:
I added a rule for docker at .github/dependabot.yml ... so it should work.